packages/storybook-html/src/components/core/typography-applying.story.mdx

```diff
@@ -0,0 +1,264 @@
+      <td>Used to indicate the term being defined within the context of a definition phrase or sentence.</td>
+    </tr>
+    <tr>
+      <td><em>Emphasis</em></td>
+      <td><code>&lt;em&gt;&lt;/em&gt;</code></td>
+      <td>Marks text that has stress emphasis. The <em> element can be nested, with each level of nesting indicating a greater degree of emphasis.</td>
+    </tr>
+    <tr>
+      <td><kbd>Shift</kbd> + <kbd>S</kbd></td>
+      <td><coded>&lt;kbd&gt;&lt;/kbd&gt;</code></td>
+      <td>Represents a span of inline text denoting textual user input from a keyboard, voice input, or any other text entry device.</td>
+    </tr>
+    <tr>
+      <td><mark>Marked text</mark></td>
+      <td><coded>&lt;mark&gt;&lt;/mark&gt;</code></td>
+      <td>Text which is marked or highlighted for reference or notation purposes, due to the marked passage's relevance or importance in the enclosing context.</td>
+    </tr>
+    <tr>
+      <td><q>Inline quotation</q></td>
+      <td><coded>&lt;q&gt;&lt;/q&gt;</code></td>
+      <td>indicates that the enclosed text is a short inline quotation. Most modern browsers implement this by surrounding the text in quotation marks.</td>
+    </tr>
+    <tr>
+      <td><s>Strikethrough</s></td>
+      <td><coded>&lt;s&gt;&lt;/s&gt;</code></td>
+      <td>renders text with a strikethrough, or a line through it. Use the <code>&lt;s&gt;</code> element to represent things that are no longer relevant or no longer accurate. However, <code>&lt;s&gt;</code> is not appropriate when indicating document edits; for that, use the del and ins elements, as appropriate.</td>
+    </tr>
+    <tr>
+      <td><samp>Output from a computer program</samp></td>
+      <td><coded>&lt;samp&gt;&lt;/samp&gt;</code></td>
+      <td>Is used to enclose inline text which represents sample (or quoted) output from a computer program.</td>
+    </tr>
+    <tr>
+      <td><small>Small text</small></td>
+      <td><coded>&lt;small&gt;&lt;/small&gt;</code></td>
+      <td>Represents side-comments and small print, like copyright and legal text, independent of its styled presentation. By default, it renders text within it one font-size smaller.</td>
+    </tr>
+    <tr>
+      <td><strong>Strong text</strong></td>
+      <td><coded>&lt;strong&gt;&lt;/strong&gt;</code></td>
+      <td>Indicates that its contents have strong importance, seriousness, or urgency.</td>
+    </tr>
+    <tr>
+      <td><sub>Subscript text</sub></td>
+      <td><coded>&lt;sub&gt;&lt;/sub&gt;</code></td>
+      <td>Specifies inline text which should be displayed as subscript for solely typographical reasons.</td>
+    </tr>
+    <tr>
+      <td><sup>Superscript text</sup></td>
+      <td><coded>&lt;sup&gt;&lt;/sup&gt;</code></td>
+      <td>Specifies inline text which is to be displayed as superscript for solely typographical reasons.</td>
+    </tr>
+    <tr>
+      <td><time>July 7</time></td>
+      <td><coded>&lt;time datetime="2018-07-07"&gt;&lt;/time&gt;</code></td>
```

Comment:
Could do with a different eg. as wouldn't know if the format is YYYY-MM-DD or YYYY-DD-MM

Reply:
its just an example ... I don;t think we need to define that here